frame/composable-traits/src/lending/math.rs

```diff
@@ -39,12 +39,10 @@ pub fn calc_utilization_ratio(
 	}
 
 	let total = cash.safe_add(&borrows)?;
-	// also max value is 1.00000000000000000, it still fails with u8, so mul by u16 and cast to u8
-	// safely
 	let utilization_ratio = borrows
 		.checked_div(&total)
 		.expect("above checks prove it cannot error")
-		.checked_mul_int(100_u16)
+		.checked_mul_int(100_u8)
```

Comment:
how that fixed issue?

Reply:
`Percent::from_percent()` requires `u8` input. `checked_mut_int()` can convert `FixedU128` to `u8` but earlier for some reason `u16->u8` conversion is done. However with new change it works and proptest also works fine. So for now should it be enough?